concrete-core/src/backends/core/private/crypto/glwe/body.rs

```diff
@@ -1,6 +1,7 @@
-use crate::math::polynomial::Polynomial;
-use crate::math::tensor::{AsMutSlice, AsMutTensor, AsRefSlice, AsRefTensor, IntoTensor, Tensor};
-use crate::tensor_traits;
+use crate::backends::core::private::math::polynomial::Polynomial;
+use crate::backends::core::private::math::tensor::{
+    tensor_traits, AsMutSlice, AsMutTensor, AsRefSlice, AsRefTensor, IntoTensor, Tensor,
+};
 
 /// The body of a GLWE ciphertext.
```

Comment:
Good catch @orelcosseron, those are not visible because they are part of the older `concrete_core` crate. I leave them for the time being, because I don't want this PR to introduce any change to the original implementation, but chances are they will be removed or made lighter in the future.